### ISSUE TITLE
DeprecationWarning in generated code with Python 3.7

### ISSUE THREAD

**Issue by dmajda** [horejsek/python-fastjsonschema#50]
In Python 3.7 with warnings enabled, `fastjsonschema.compile()` (or rather the code generated by it) emits `DeprecationWarning`s when using `pattern` or `patternProperties` with some regexps:

```console
$ python3 --version
Python 3.7.0
$ python3 -Wall -c 'import fastjsonschema; fastjsonschema.compile({"pattern": r"\d"})'                
<string>:6: DeprecationWarning: invalid escape sequence \d
$ python3 -Wall -c 'import fastjsonschema; fastjsonschema.compile({"patternProperties": {r"\d":{}}})'
<string>:8: DeprecationWarning: invalid escape sequence \d
```

It would be good if the generated code was warning-free on Python 3.7. (In our specific case, this warning pollutes our test suite output.)

I can probably prepare a patch, this looks like a simple escaping issue.
